### DISCUSSION TITLE
New feature request : add support of libravatar

### DISCUSSION THREAD

**Discussion by paulolesgaz** [dani-garcia/vaultwarden#2663] [Ideas]
Hello, actually vaultwarden support the display of avatars through Gravatar, which is great. But could you add also the possibility to use libravatar, which does the same thing as Gravatar, but is open source ?

Many thanks for your great work. 

Vaulwarden for ever ! ❤️

**Comment by BlackDex** [comment]
I'm afraid that isn't that easy. Since this is built-in into the client, and had no server side items at all. Soz this feature request would probably be better for upstream bitwarden/clients

**Comment by boospy** [comment]
Does Gravatar support even exist anymore? Because I just installed a fresh Vaultwarden server, but I can't find the feature.

**Reply by BlackDex** [reply]
That has been removed for a while now because of privacy concerns

**Comment by boospy** [comment]
Thank you for that information.
